static/js/todo_widget.js

```diff
@@ -154,6 +154,12 @@ exports.activate = function (opts) {
             const data = task_data.handle.new_task.outbound(task, desc);
             callback(data);
         });
+
+        elem.find("input#toggle-completed").on("click", (e) => {
```

Comment:
This code assumes that the checkbox and the other elements never get out of sync, which seems like a reasonable assumption, but it might be less brittle to say...

~~~
if (checked) {
  foo.hide();
  bar.hide();
else {
  foo.show();
  foo.show();
}